website/docs/timeline.md

```diff
@@ -3,40 +3,384 @@ title: Timeline
 toc: true
 ---
 
-## Timeline
-At its core, Hudi maintains a `timeline` of all actions performed on the table at different `instants` of time that helps provide instantaneous views of the table,
-while also efficiently supporting retrieval of data in the order of arrival. A Hudi instant consists of the following components
+A Hudi table maintains all operations happened to the table in a single timeline comprised of two parts, an active timeline and an archived timeline. The active timeline stores all the recent instant, while the archive timeline stores the older instants. An instant is a transaction where all respective partitions within a base path have been successfully updated by either a writer or a table service. Instants that get older in the active timeline are moved to archived timeline at various times.
```

Comment:
`all the recent instant` -> `all the recent instants`

Reply:
`the archive timeline stores` -> `the archived timeline stores`

Reply:
good catch- thanks!